client/src/Lenkeflis/Lenkeflis.tsx

```diff
@@ -11,20 +13,44 @@ export const Lenkeflis: React.FunctionComponent<{
   href: string | undefined;
   fyltoppBakgrunn?: boolean;
 }> = ({ overskrift, brødtekst, ikon, href, fyltoppBakgrunn }) => {
+  const router = useRouter();
+  const TIMEOUT_IN_MILLIS = 750;
+
+  const håndtereOnClickMedTimeout = (
+    destinasjon: string,
+    lenketekst: string,
+    timeout: number
+  ) => {
+    setTimeout(() => {
+      router.push(destinasjon);
+    }, timeout);
```

Comment:
For noen som ikke vet context, så vil denne timeouten være ganske uforståelig. Kan vi navngi ting, evt legge inn en kommentar, som bedre forklarer hvorfor vi trenger timeouten?